jupyterlab.tex

```diff
@@ -36,5 +35,5 @@ \subsection{Case Study: Deploying releases to JupyterLab}
 kubernetes YAML, a rewrite so that deployment becomes a Terraform
 configuration rather than a python module, and/or integration of TLS
 certificate management into the tool.
-
-A more detailed description of the LSST JupyterLab environment can be found in Ref.~\citenum{AS18-AS110-10707-16}.
+%%WOM Last line could be reomved.
```

Comment:
Yes. Although I'd like to cite @frossie's JupyterLab SPIE paper somewhere in the paragraph.